### ISSUE TITLE
pruning strategy code

### ISSUE THREAD

**Issue by junho2000** [zuosc19/QuadricFormer#9]
Hello!

Thank you for making this great work, this inspired me a lot.

BTW i want to look pruning, splitting, refine code. Where can i find it?

**Comment by Hanxuan411** [comment]
> Hello!
> 
> Thank you for making this great work, this inspired me a lot.
> 
> BTW i want to look pruning, splitting, refine code. Where can i find it?

I also have the same question. Have you found the code for this part?

**Comment by GCChen97** [comment]
It is already 2026. Has anyone found this so-called 'prune-and-split' module anywhere else besides the accepted paper?

**Comment by Starlento** [comment]
Same question. And curious why the training command in readme is using validation set, maybe we cannot reproduce the checkpoint?

**Comment by Bruce-WangGF** [comment]
> Same question. And curious why the training command in readme is using validation set, maybe we cannot reproduce the checkpoint?

Their papers look like training in the training set from nuscenes, and test in the validation set, except the real testing set. GaussianFormer and GaussianFormerv2 have the same problems.

**Comment by GCChen97** [comment]
> > Same question. And curious why the training command in readme is using validation set, maybe we cannot reproduce the checkpoint?
> 
> Their papers look like training in the training set from nuscenes, and test in the validation set, except the real testing set. GaussianFormer and GaussianFormerv2 have the same problems.

In the paper of SurroundOcc from their lab, they said "Since the 3D semantic and 3D detection labels are unavailable in test set and we cannot generate dense occupancy labels, we use the training set to train the model and validation set for evaluation.". This might explain why the val set was used for evaluation.

Do you find that other works do evaluations on the real test set?

**Comment by Bruce-WangGF** [comment]
> > > Same question. And curious why the training command in readme is using validation set, maybe we cannot reproduce the checkpoint?
> > 
> > 
> > Their papers look like training in the training set from nuscenes, and test in the validation set, except the real testing set. GaussianFormer and GaussianFormerv2 have the same problems.
> 
> In the paper of SurroundOcc from their lab, they said "Since the 3D semantic and 3D detection labels are unavailable in test set and we cannot generate dense occupancy labels, we use the training set to train the model and validation set for evaluation.". This might explain why the val set was used for evaluation.
> 
> Do you find that other works do evaluations on the real test set?

Get it! Thank you. I haven’t seen any.
